modules/server/templates/scripts/forseti-server/forseti_server_startup_script.sh.tpl

```diff
@@ -6,6 +6,10 @@ USER=ubuntu
 USER_HOME=/home/ubuntu
 INTERNET_CONNECTION="$(ping -q -w1 -c1 google.com &>/dev/null && echo online || echo offline)"
 
+export USER_HOME
+
+INIT_SERVICES_MD5_HASH=${forseti_run_forseti_services_md5_hash}
```

Comment:
For the sake of clarity, I would rename this variable to something like `RUN_FORSETI_SERVICES_MD5_HASH`

Reply:
My bad, renamed.